### PR TITLE
PLAT-395: integration test for case 4974

### PR DESCRIPTION
**Pull request by MariaVasilenko**
**- What I did**
added new integration test

**- How to verify it**
run TestVirtual_Method_WithExecutor_ObjectIsNotExist

### CODE REVIEW

File: ledger-core/v2/virtual/integration/method_test.go [hunk 1]
```diff
@@ -164,6 +164,57 @@ func TestVirtual_Method_WithoutExecutor(t *testing.T) {
...
+		requestIsDone := make(chan struct{}, 0)
+
+		server.PublisherMock.SetChecker(func(topic string, messages ...*message.Message) error {
+			defer func() { requestIsDone <- struct{}{} }()
+
+			pl, err := payload.UnmarshalFromMeta(messages[0].Payload)
+			require.NoError(t, err)
+
+			switch pl.(type) {
+			case *payload.VCallResult:
```
Comment (MariaVasilenko) [inline ledger-core/v2/virtual/integration/method_test.go:200]: i have *payload.VStateRequest type in pl

File: ledger-core/v2/virtual/integration/method_test.go [hunk 2]
```diff
@@ -108,7 +108,7 @@ func Method_PrepareObject(ctx context.Context, server *utils.Server, class refer
 	}
 }
 
-func TestVirtual_Method_WithoutExecutor(t *testing.T) {
+func TestVirtual_Method_WithoutExecutor(t *testing.T) { // todo why the test have a name "without executor"?
```
Comment (ruz) [inline ledger-core/v2/virtual/integration/method_test.go:111]: remove this todo question from PR. "WithoutExecutor" indicates that "executor of builtin contracts" is mocked.

File: ledger-core/v2/virtual/integration/method_test.go (hunk 1 above)
Comment (ruz) [reply ledger-core/v2/virtual/integration/method_test.go:200]: you should fill objects catalog with a state of the object to avoid VStateRequest request

**Comment by MariaVasilenko** [comment]
moving to new branch

**Pull request closed without merging**